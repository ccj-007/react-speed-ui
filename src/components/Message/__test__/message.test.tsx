import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Message } from '../../index';

describe('test Message component', () => {
  it('should render', () => {
    const { asFragment } = render(<Message type='success' />);
    expect(asFragment()).toMatchSnapshot();
  });
});
