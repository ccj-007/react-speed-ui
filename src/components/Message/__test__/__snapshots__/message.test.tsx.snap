// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Message component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-message speed-message-success"
  >
    <div
      class="speed-message-content"
    >
      <div
        class="speed-message-icon"
      />
      <div />
    </div>
  </div>
</DocumentFragment>
`;
